libra-dev/src/account_resource.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_types::{
+    account_config::get_account_resource_or_default,
+    account_state_blob::AccountStateBlob,
+    //    byte_array::ByteArray,
+};
+use std::slice;
+
+#[derive(Debug)]
+pub struct DevAccountResource {
```

Comment:
We probably don't need two copies of the struct, the C one should be all we need.